docs/copiloting-service.md

```diff
@@ -0,0 +1,119 @@
+# OpenSAFELY copiloting service
+
+## What is the OpenSAFELY copiloting service?
+
+All new users of the OpenSAFELY platform are offered access to our copilot service, which pairs the new user (the "pilot") with an experienced OpenSAFELY researcher (the "copilot") to help with the onboarding process. The objective of this service is to help new users get up and running with OpenSAFELY as quickly as possible and aims to facilitate the generation of analytical outputs from real data within a four week period.
+
+The service includes:
+- dedicated 1:1 support with cohort extraction via the OpenSAFELY plaform
+- community support via GitHub forums and the #opensafely-users Slack channel
+- GitHub introduction and coaching
+- output checking service 
+- manuscript review
+- opportunities to present at quarterly OpenSAFELY User Group meetings
```

Comment:
Formatting has not worked here in the preview https://2aa8b5c6.opensafely-docs.pages.dev/copiloting-service/

Reply:
Thanks, amended.